indico/modules/events/registration/controllers/__init__.py

```diff
@@ -46,7 +46,8 @@ def _process(self):
         form = make_registration_form(self.regform, management=self.management, registration=self.registration)()
 
         if form.validate_on_submit():
-            modify_registration(self.registration, form.data, management=self.management)
+            notify_user = request.form.get('send_modification_notification') == 'y'
```

Comment:
Probably you can do something like this in there:

```
    class RegistrationFormWTF(IndicoForm):
        if management and registration:
            notify_modification = BooleanField(...)
```

If not just do `RegistrationFormWTF.notify_modification = BooleanField(...)` later in the functio